Python/getargs.c

```diff
@@ -613,20 +613,19 @@ convertitem(PyObject *arg, const char **p_format, va_list *p_va, int flags,
 /* Format an error message generated by convertsimple(). */
 
 void
-_PyArg_BadArgument(const char *fname, int iarg,
+_PyArg_BadArgument(const char *fname, int iarg, const char *argname,
                    const char *expected, PyObject *arg)
 {
-    if (iarg) {
```

Comment:
Don't miss a special case of a single argument.

Reply:
Thanks, I had not correctly understood how argument clinic was working, I see now that iarg is 0 iif there is a single positional argument. I will put the old behavior back.